openlattice.yaml

```diff
@@ -2825,6 +2825,30 @@ paths:
                 type: object
                 additionalProperties:
                   $ref: "#/components/schemas/Auth0userBasic"
+  /datastore/principals/users/search/"{searchQuery}":
```

Comment:
are these quotes supposed to be here?

Reply:
They are.  If we plug in a searchQuery in R or python, it will not explicitly put quotes around it, and can lead to problematic URL's (spaces etc).